consumers/google.go

```diff
@@ -247,6 +281,18 @@ func (d *googleDNSConsumer) printRecords(records map[string]*ownedRecord) {
 	}
 }
 
+func (d *googleDNSConsumer) hostedZoneFor(name string) string {
```

Comment:
there is a similar `func` in AWS consumer (not namespaced, because I thought it is something we would use) . How about we modify it a little bit and place in `consumers.go` ? I am fine with keeping it like this for now too and refactoring later ;)